client/src/endpoint/account.rs

```diff
@@ -452,3 +471,175 @@ mod effects_tests {
         assert_eq!(req.uri().query(), Some("cursor=CURSOR&order=asc&limit=123"));
     }
 }
+
+/// Represents the payments for account endpoint on the stellar horizon server.
+/// The endpoint will return all the Payment operations where a specific account
+/// is either the sender or receiver.
+///
+/// <https://www.stellar.org/developers/horizon/reference/endpoints/payments-for-account.html>
+///
+/// ## Example
+/// ```
+/// use stellar_client::sync::Client;
+/// use stellar_client::endpoint::{account, payment};
+///
+/// let client = Client::horizon_test().unwrap();
+///
+/// // Grab payments and associated account to ensure an account with payments
+/// let all_payments = client.request(payment::All::default().limit(1)).unwrap();
+/// let payment      = &all_payments.records()[0];
+/// let account_id   = payment.from();
```

Comment:
@kbacha @correlator I struggled with the best way to get an account ID from the `payment::All` response. The response struct are generic `Operation` types. The associated account IDs appear contained in the `kind` attribute child struct, but not all Operations have some account identifier and they are not keyed consistently across operation types from what I can grok. 

What would be the Rust Way to get the account ID out of these Operations?

Reply:
Something like this may work:

```rust
let account_id = match payment.kind() {
    OperationKind::Payment(payment) => payment.from(),
    _ => panic!("Expected payment type"),
};
```